### PR TITLE
- Update on Assetic Version

### PR DESCRIPTION
**Pull request by solcre**
Tests are passing, and my test site is working fine. I would definitely update version since it has a couple of fixes, the one that got my attention was on a StringAsset when writing a test.

### CODE REVIEW

**Comment by RWOverdijk** [comment]
:+1:. @Ocramius?

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "zendframework/zend-loader": "2.*",
         "zendframework/zend-eventmanager": "2.*",
         "zendframework/zend-mvc": "2.*",
-        "kriswallsmith/assetic": "~1.1.2"
+        "kriswallsmith/assetic": "1.2.1"
```
Comment (Ocramius) [inline composer.json]: Please use `~1.2.1`

**Comment by solcre** [comment]
done!

**Comment by Ocramius** [comment]
:+1:

**Pull request merged**